### PR TITLE
Correction on quarkus-platform job

### PR DESCRIPTION
**Pull request by radtriste**

### CODE REVIEW

File: .ci/jenkins/Jenkinsfile.update-quarkus-platform [hunk 1]
```diff
@@ -36,70 +29,75 @@ pipeline {
...
+
+                        githubscm.pushObject('--force-with-lease origin', localBranch, "${FORK_GIT_AUTHOR_CREDS_ID}")
                     }
                 }
             }
         }
     }
 }
 
+def squashCommits(String baseBranch, String newCommitMsg) {
```
Comment (radtriste) [inline .ci/jenkins/Jenkinsfile.update-quarkus-platform]: will use shared libraries once https://github.com/kiegroup/jenkins-pipeline-shared-libraries/pull/221 is merged

**Comment by radtriste** [comment]
tested in 
- https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/custom/job/tradisso/job/update-quarkus-platform-kogito/
- https://eng-jenkins-csb-business-automation.apps.ocp-c1.prod.psi.redhat.com/job/custom/job/tradisso/job/update-quarkus-platform-optaplanner/

**Comment by radtriste** [comment]
and created PR: https://github.com/quarkusio/quarkus-platform/pull/659

**Comment by radtriste** [comment]
depends on https://github.com/kiegroup/jenkins-pipeline-shared-libraries/pull/221

**Review by mbiarnes — APPROVED** [review]
LGTM

**Comment by radtriste** [comment]
all good. removing test config commit and merging

**Pull request merged**